modules/activemq/activemq.go

```diff
@@ -29,10 +29,11 @@ var (
 )
 
 var (
-	defMaxQueues   = 50
-	defMaxTopics   = 50
-	defURL         = "http://127.0.0.1:8161"
-	defHTTPTimeout = time.Second
+	defMaxQueues       = 50
```

Comment:
`def` usually means not `default` but `define`.
I think use `default` is more readable.